Lesson-7/index.html

```diff
@@ -0,0 +1,184 @@
+    </div>
+    <div class="apartment_sunshine">
+    <img class="sunshine_img" src="image/sunshine.png" alt="sunshine_img">
+    <p class="text_apartment_sunshine">Apartment Sunshine</p>
+    <p class="location_sunshine">Santa Cruz de Tenerife, Spain</p>
+    </div>
+    <div class="villa_kunerad">
+    <img class="kunerad_img" src="image/kunerad.png" alt="kunerad_img">
+    <p class="text_villa_kunerad">Villa Kunerad</p>
+    <p class="location_kunerad">Vysokie Tatry, Slowakia</p>
+    </div>
+    <div class="hostel_friendship">
+    <img class="frendship_img" src="image/frendship.png" alt="frendship_img">
+    <p class="text_hostel_friendship">Hostel Friendship</p>
+    <p class="location_frendship">Berlin, Germany</p>
+    </div>
+    <button class="slider-arrows">></button>
+    </div>
+    </div>
+    </section>
+
+    <section class="The_best_destinations">
+        <h2 class="The_best_destinations_heading">The best destinations</h2> 
+        <div class="seat_selection_buttons">
+            <input class="Regions" type="button" value="Regions" placeholder="Regions">
+            <input class="Cities" type="button" value="Cities" placeholder="Cities">
+            <input class="Places" type="button" value="Places of interest" placeholder="Places of interest">
+        </div>
+        <div class="countries">
+            <div class="Maldives">
+                <img src="image/Maldives.svg" alt="Maldives" class="Maldives_img">
+                <p class="Maldives_text">Maldives</p>
+                <input class="Maldives_book_now" type="button" value="Book now" placeholder="Book now">
+            </div>
+            <div class="Spain">
+                <img src="image/Spain.svg" alt="Spain" class="Spain_img">
+                <p class="Spain_text">Spain</p>
+                <input class="Spain_book_now" type="button" value="Book now" placeholder="Book now">
+            </div>
+            <div class="Norway">
+                <img src="image/Norway.svg" alt="Norway" class="Norway_img">
+                <p class="Norway_text">Norway</p>
+                <input class="Norway_book_now" type="button" value="Book now" placeholder="Book now">
+            </div>
+            <div class="Island">
+                <img src="image/Island.svg" alt="Island" class="Island_img">
+                <p class="Island_text">Island</p>
+                <input class="Island_book_now" type="button" value="Book now" placeholder="Book now">
+            </div>
+        <button class="arrow_to_down">^</button>
+        </div>
+        </section>
+
+        <section class="sign_up">
+            <div class="discount">
+        <img class="user_img" src="image/user.png" alt="user_img">
+<div class="container_sign_up">
+<label class="sign_up_text" for="button">Sign up and get 20% off your first booking</label>
+<input class="button_sing_up" type="button" value="Sign up" placeholder="Sign up">
+</div>
+            </div>
+            <a href="#" class="close"></a>
```

Comment:
bad naming.
:hover working incorectly 
![image](https://user-images.githubusercontent.com/20189313/129492102-e1bcb8ad-0918-497f-8f01-d5a72b6c5d38.png)

Reply:
should be button